lib/view_models/user_provider.dart

```diff
@@ -36,6 +36,7 @@ class UserProvider extends BaseModel {
     setStatus(UPDATE_USER, Status.Loading);
     try {
       var atKeys = await AtKeySetService().getAtkeys();
+      ;
```

Comment:
@Pritam-deb I think this is unwanted.

Reply:
That was a mistake I didn't observe. Will remove it.